package.xml

```diff
@@ -80,6 +80,22 @@
  <providesextension>mysqlnd_azure</providesextension>
  <extsrcrelease />
  <changelog>
+    <release>
+     <version>
+     <release>1.1.1</release>
+     <api>1.1.1</api>
+     </version>
+     <stability>
+      <release>stable</release>
+      <api>stable</api>
+     </stability>
+     <date>2020-05-13</date>
+     <license uri="http://www.php.net/license">PHP License</license>
+     <notes>
+- 1. New connection attribute "_extension_version", indicate the mysqlnd_azure extension build version.
```

Comment:
The notes also need update in  section above, where will actually be shown in peck package. Check line 30

Reply:
@coko156